test/AtomicSwapEther.js

```diff
@@ -0,0 +1,319 @@
+const chai = require("chai");
+const chaiAsPromised = require("chai-as-promised");
+const { expectEvent } = require("openzeppelin-test-helpers");
+
+const { getBalance, makeRandomAddress, makeRandomID, makeTimeout, sleep } = require("./utils");
+
+const atomicSwap = artifacts.require("./AtomicSwapEther.sol");
+
+const { expect } = chai;
+chai.use(chaiAsPromised);
+
+contract("AtomicSwapEther", accounts => {
+  const defaultPreimage = "0x42a990655bffe188c9823a2f914641a32dcbb1b28e8586bd29af291db7dcd4e8";
+  const defaultHash = "0x261c74f7dd1ed6a069e18375ab2bee9afcb1095613f53b07de11829ac66cdfcc";
+  const nullPreimage = "0x0000000000000000000000000000000000000000000000000000000000000000";
+  const defaultAmount = 50e6;
+
+  let testContract;
+
+  before(async () => {
+    testContract = await atomicSwap.deployed();
+  });
+
+  describe("open()", () => {
+    it("accepts ether", async () => {
+      const id = makeRandomID();
```

Comment:
Could we use strict camelCase notation for acronyms as well? Otherwise we end up with stuff like `myHTTPJSONAPI` at some point.